### PR TITLE
Update French 1

### PR DESCRIPTION
**Pull request by bubuleur**

### CODE REVIEW

File: app/src/main/res/values-fr/arrays.xml [hunk 1]
```diff
@@ -1,14 +1,57 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <string-array name="fps_names">
+        <item>30 IPS</item>
+        <item>60 IPS</item>
+        <item>90 IPS</item>
+        <item>120 IPS</item>
+    </string-array>
+    <string-array name="language_names" translatable="false">
```
Comment (cgutman) [inline app/src/main/res/values-fr/arrays.xml:9]: You should not translate the ones that say `translatable="false"`. Those should be left out of this file.

Comment (bubuleur) [reply app/src/main/res/values-fr/arrays.xml]: Ok I didn't know sorry it's Okay I changed why not translate them ?

Comment (cgutman) [reply app/src/main/res/values-fr/arrays.xml]: They are not displayed to the user. They are used for internal purposes.

**Comment by cgutman** [comment]
@bubuleur I added some new values in array.xml for surround sound. Can you translate those too?

**Pull request merged**